### PR TITLE
update logging doc

### PR DESCRIPTION
**Pull request by suoning**
logging.md error.

config.py:
LOGGING = {
    ...
    'handlers': {
        ...
        'accessTimedRotatingFile': {
            'class': 'logging.handlers.TimedRotatingFileHandler',
            'filters': ['accessFilter'],
            'formatter': 'access',
            'when': 'D',
            'interval': 1,
            'backupCount': 7,
            'filename': 'access.log'
        },
        '**errorTimedRotatingFile**': {
            'class': 'logging.handlers.TimedRotatingFileHandler',
            'filters': ['errorFilter'],
            'when': 'D',
            'interval': 1,
            'backupCount': 7,
            'filename': 'error.log',
            'formatter': 'simple'
        }
    },
    'loggers': {

    }
}

### CODE REVIEW

File: docs/sanic/logging.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ from sanic.config import LOGGING
 # The default logging handlers are ['accessStream', 'errorStream']
 # but we change it to use other handlers here for demo purpose
 LOGGING['loggers']['network']['handlers'] = [
-    'accessTimedRotatingFile', 'errorTimedRotationgFile']
+    'accessTimedRotatingFile', 'errorTimedRotatingFile']
```
Comment (messense) [inline docs/sanic/logging.md]: Can you change these two to `accessSyslog` and `errorSyslog`?

The file loggers were removed in #706

Comment (messense) [inline docs/sanic/logging.md]: I forget to changes these at that time...

File: docs/sanic/logging.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ from sanic.config import LOGGING
 # The default logging handlers are ['accessStream', 'errorStream']
 # but we change it to use other handlers here for demo purpose
 LOGGING['loggers']['network']['handlers'] = [
-    'accessTimedRotatingFile', 'errorTimedRotationgFile']
+    'accessSysLog', 'errorSysLog']
```
Comment (messense) [inline docs/sanic/logging.md]: Needs to update https://github.com/channelcat/sanic/pull/724/files#diff-7e3514b34ffada2985c8db3b21bf4794R29 too.

Comment (suoning) [inline docs/sanic/logging.md]: Is it ok to delete this sentence?

Comment (messense) [inline docs/sanic/logging.md]: I think it's ok.

**Comment by r0fls** [comment]
@messense is this good to go now?

**Pull request merged**